src/explore-education-statistics-common/src/components/FigureFootnotes.tsx

```diff
@@ -1,18 +1,18 @@
 import CollapsibleList from '@common/components/CollapsibleList';
+import { Footnote } from '@common/services/types/footnotes';
 import React from 'react';
+import VisuallyHidden from './VisuallyHidden';
 
 interface Props {
-  footnotes: {
-    id: string;
-    label: string;
-  }[];
+  footnotes: Footnote[];
+  hiddenText?: string;
```

Comment:
Would name this `headingHiddenText` to avoid ambiguity over what this relates to.